### PR TITLE
How CART algorithms are substantial in speech emotion recognition

### LINKED ISSUE #4674: [Artificial Intelligence] How CART algorithms are substantial in speech emotion recognition
#### Proposed article introduction
Speech emotion recognition is vital in enhancing the efficiency and naturalness of the interaction between humans and computers.
The CART algorithm is a crucial concept of artificial intelligence that could be used in data exploration
The study focuses on assessing the role of CART algorithms in speech emotion recognition
#### Key takeaways 
- Why is speech emotion recognition meaningful?
- How are CART algorithms effective in speech emotion recognition?
- For example, how are CART algorithms meaningful in lowering computational complexity?
#### Article quality
The article is unique and focuses on one of the critical untapped areas of artificial intelligence, specifically enhancing the naturalness and efficiency of human-computer interaction. The articles go in-depth to assess how effective CART algorithms can assess speech emotion recognition. 
#### References
Not applicable

### PR DESCRIPTION
**Pull request by slynn22**
This closes #4674

### CODE REVIEW

**Comment by WanjaMIKE** [comment]
Plagiarism check done

**Comment by collins-a** [comment]
Hello @slynn22. Welcome to EngEd! As a new contributor, I encourage you to add a brief Author Bio (index.md file) and Headshot image (avatar file). Include these files in the **authors** folder within the GitHub repo via your forked repo similar to the article files so as to be credited for your hard work.

**Comment by slynn22** [comment]
@collins-a , i have revised as requested

**Comment by collins-a** [comment]
> @collins-a , i have revised as requested

Hey @slynn22, thank you for adding the files. Please edit the index.md author file as per the format outlined at the end of this [page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/UPLOAD_INSTRUCTIONS.md), to reflect the required fields.

**Comment by slynn22** [comment]
@collins-a , i have updated as requested

File: content/articles/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/hero.jpg
+    alt: CART example image 
+---
+Unlike humans, who have the human ability to recognize emotion from speech, machines do not have the sense to analyze feelings. Speech emotional recognition is vital in understanding human emotions such as amusements, frustrations, or disappointment.
+<!--more-->
+
+Emotions are vital in day-to-day life, especially in interpersonal human interaction. They are helpful both in intelligent and rational decisions. Unlike humans, who have the human ability to recognize emotion from speech, machines do not have the sense to analyze feelings. Speech emotional recognition is vital in understanding human emotions such as amusements, frustrations, or disappointment. Speech emotion recognition informs the computer on the physical and emotional state of an individual from a speech.
```
Comment (collins-a) [inline content/articles/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/index.md:20]: Consider changing the listed emotions to singular form as it's more grammatically suitable.

File: content/articles/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+A wide range of studies have focused on detecting a person's emotions. They include the use of computer games, audio surveillance, robots, others. Machine learning algorithms have been termed as more efficient in classification and prediction, which is vital in speech emotion recognition. However, there is a lot of information about the use of [CART](/engineering-education/decision-tree-in-python/) in speech emotion recognition that needs assessment which the study focuses on.
+
+The article focuses on assessing the substantial use of CART algorithms in speech emotion recognition. In achieving the purpose, the article reviews the speech emotion system, how CART works, and various reasons affiliated with its effectiveness.
+
+### The system of emotion recognition
+
+The system of speech emotion recognition is so like the pattern system. The system automatically identifies the human beings' emotions from a voice. The system is usually based on the signal of speech and emotion recognition methods. The system consists of five main steps;
```
Comment (collins-a) [inline content/articles/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/index.md:28]: Rephrase the first sentence to better convey the comparison between speech emotion recognition and the pattern system

File: content/articles/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/index.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+The various emotional recognition extracts critical emotional features from speech. Pitch is one of the essential features, which is the sound's frequency. The features also include standard deviation, energy intensity, and shimmer. Others include zero-crossing rate, short-time energy autocorrelation, and [MFCC](https://musicinformationretrieval.com/mfcc.html#) (Mel frequency cepstral coefficient). Other features include spectral roll-off, spectral flux, noise to harmonic ratio, harmonics to noise ratio, and jitter.
+
+#### Training
+
+This is another crucial step of the speech recognition system. The step entails voice training of the computers for speech recognition. The entails entail assigning a weight of features using estimators and considering small sets of features. The estimators are trained on the features of the initial stage and predictive power.
```
Comment (collins-a) [inline content/articles/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/index.md:48]: The second line _the entails entail...._. Rephrase this

**Review by collins-a — CHANGES_REQUESTED** [review]
@slynn22 great first article. Handle the minor revisions, and perform a Grammarly check on your article, then ping me when done.

File: content/articles/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/index.md (hunk 3 above)
Comment (slynn22) [reply content/articles/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/index.md:48]: Alright, will handle them

**Comment by slynn22** [comment]
@collins-a

**Comment by hectorkambow** [comment]
@collins-a  

If any more edits are needed lets consider closing the PR to allow the author the proper time to make any needed changes. 

👍

**Comment by collins-a** [comment]
> @collins-a
> 
> If any more edits are needed lets consider closing the PR to allow the author the proper time to make any needed changes.
> 
> 👍

@slynn22 upon going through your article again, I've noted a few grammatical errors, showing you failed to carry out the Grammarly check as advised. As such, due to the increasing rounds of revision, we'll be closing this PR to allow you to submit the article once more when it's publish-ready.

Make sure to ping me once you submit the PR to expedite the revision before the final review. Good luck.

**Comment by slynn22** [comment]
@collins-a, this is the new PR of the order which you asked me to revise (#4887)

**Pull request closed without merging**